yarn.lock

```diff
@@ -3,21 +3,21 @@
 
 
 "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.14.5":
```

Comment:
Why is there such a big diff here?

Reply:
That's a good question... I tried with Node 12 and Node 14 and every time there was a huge difference.
What Node version do you use?

Reply:
The diff looks fine, the dependabot PRs also have a massive diff so idk what changed but clearly something did.

Reply:
yeah, that creates merge conflicts... I need to find out how to solve that...
Do you have this big difference as well?

Reply:
I fixed the conflicts. I must have used `npm i` in the past which created those issues. It should be good now, Axios is added to yarn.lock.